assets/helpers/contributions.js

```diff
@@ -5,6 +5,7 @@
 import { roundDp } from 'helpers/utilities';
```

Comment:
Should we merge this and [this file](https://github.com/guardian/support-frontend/blob/master/assets/components/contributionSelection/helpers.js#L18)? Both looks like are helpers for the act of contributing.

Reply:
The `helpers` file in `contributionSelection` has a lot of presentation-specific functions that I simply pulled out of the main component to avoid cluttering that file. The `contributions` helper is more for shared logical code that can be used regardless of the presentational aspect. It's possible that a couple of functions could be shared, but not the whole file.

Reply:
4af563d